_posts/2016-03-29-1.5.0-release.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title:  "Here comes 1.5.0 !"
+date:   2016-03-23 10:00:00
+categories: annonce
+---
+
+- Features:
+  - The SDK now needs an initialization key to contact the Hive, [See there](/en/1.5.0/guide/dashboard/#handle-private-crop).
+  - You can now upload media ([video](/en/1.5.0/stings/form/formPage.html#video), [audio](/en/1.5.0/stings/form/formPage.html#sound), and [pictures](/en/1.5.0/stings/form/formPage.html#picture)) from _form_.
+  - Needed runtime permissions for _Android M_ are now declared per Sting, you can see required and missing runtime permissions for each crop.
+  - You can write scripts to process each uploaded data block before insertion in database. [More info over here](/en/1.5.0/guide/dashboard/#filter-uploaded-data).
+- Enhancement:
+  - The SDK will now automatically log out users which session has become invalid.
```

Comment:
"The SDK will now automatically log out users with invalid session" might be better imo
(not sure about the "which")